Dockerfile

```diff
@@ -151,33 +112,25 @@ RUN pip3 install --editable /nextstrain/augur
 # used for the same reasons described above.
 RUN cd /nextstrain/auspice && npm install && npm run build && npm link
 
+RUN apt-get clean
```

Comment:
Because each directive in Dockerfile creates a new layer, this `apt-get clean` will remove apt's cache files from the layer which it creates, but the layers before it will still contain the files and still must be downloaded with the rest of the layers. The typical solution would be to bundle the `apt-get clean` in the same `RUN` as the `apt-get install`, but the Debian image this is ultimately based on helpfully pre-configures apt to run what's effectively `apt-get clean` after every install:

https://github.com/debuerreotype/debuerreotype/blob/6b722a49935a94a9f718f07616f0863db6267023/scripts/debuerreotype-minimizing-config#L85-L109

Which is all to say, this line isn't working as intended and I think can just be removed. :-)

Reply:
Awesome! Thank you for digging that up from the Python image! I've removed these lines.

Reply:
:-) I was pretty sure the base image did this auto-cleaning, but couldn't find it documented, so I dug into the source quickly to make sure I wasn't making things up.